automated-comments/java/two-fer/avoid_string_format.md

```diff
@@ -0,0 +1,7 @@
+Although `String.format` is a wonderful and powerful method, it is extremely
+slow for a problem that only requires simple String concatenation with the `+`
```

Comment:
```suggestion
slow for a problem that only requires simple `String` concatenation with the `+`
```

Reply:
Is it extremely slow?  How do we keep track of this when/if that ever gets changed or improved?  What is the threshold from extremely slow to just plain slow?  And when do we update this information?